spec/features/access_requests_spec.rb

```diff
@@ -127,33 +152,6 @@
       expect(page).to have_text("Open access requests")
     end
 
-    it "lower-cases the recipient email because the API is case-sensitive" do
```

Comment:
Are we getting rid of this because we no longer have the email?

Reply:
This is handled and tested on the backend

`target_user = User.find_or_create_by!(email: @access_request.email_address.downcase)`